Makefile

```diff
@@ -1,7 +1,7 @@
 HUB ?= quay.io/maistra-dev
 
 BUILD_IMAGE = maistra-builder
-BUILD_IMAGE_VERSIONS = $(BUILD_IMAGE)_2.1 $(BUILD_IMAGE)_2.0 $(BUILD_IMAGE)_1.1 ${BUILD_IMAGE}_1.0
+BUILD_IMAGE_VERSIONS = $(BUILD_IMAGE)_2.1 $(BUILD_IMAGE)_2.0
```

Comment:
Do you still want to keep 1.1 and 1.0 files around?

Reply:
good point, I removed them. now I also don't need to make that noop change in the Dockerfile